test/mailers/patient_mailer_test.rb

```diff
@@ -290,14 +372,43 @@ def twilio_error
       true
     end)
     expect_any_instance_of(::Twilio::REST::Studio::V1::FlowContext::ExecutionList).to(receive(:create)).with({
-                                                                                                               from: 'test',
                                                                                                                to: '+15555550111',
-                                                                                                               parameters: params
+                                                                                                               parameters: params,
+                                                                                                               from: 'test_messaging_sid'
                                                                                                              })
     PatientMailer.assessment_sms(@patient).deliver_now
   end
 
-  test 'assessment voice message content' do
+  test 'assessment sms message content not using messaging service' do
+    ENV['TWILLIO_MESSAGING_SERVICE_SID'] = nil
+    dependent = create(:patient)
+    dependent.update(responder_id: @patient.id, submission_token: SecureRandom.hex(20))
+
+    params = {
+      language: 'EN',
+      try_again: I18n.t('assessments.sms.prompt.try-again', locale: 'en'),
+      thanks: I18n.t('assessments.sms.prompt.thanks', locale: 'en'),
+      medium: 'SMS',
+      patient_submission_token: @patient.submission_token,
+      # Don't have any symptoms set up for this jurisdiction.
+      threshold_hash: @patient.jurisdiction.jurisdiction_path_threshold_hash,
+      # rubocop:disable Layout/LineLength
+      prompt: "#{I18n.t('assessments.sms.prompt.daily1', locale: 'en')}-0, -0.#{I18n.t('assessments.sms.prompt.daily2-p', locale: 'en')}#{I18n.t('assessments.sms.prompt.daily3', locale: 'en')}.#{I18n.t('assessments.sms.prompt.daily4', locale: 'en')}"
+      # rubocop:enable Layout/LineLength
+    }
+
+    allow_any_instance_of(::Twilio::REST::Studio::V1::FlowContext::ExecutionList).to(receive(:create) do
+      true
+    end)
+    expect_any_instance_of(::Twilio::REST::Studio::V1::FlowContext::ExecutionList).to(receive(:create)).with({
+                                                                                                               to: '+15555550111',
+                                                                                                               parameters: params,
+                                                                                                               from: 'test'
+                                                                                                             })
+    PatientMailer.assessment_sms(@patient).deliver_now
+  end
+
+  test 'assessment voice message content using messaging service' do
```

Comment:
Didn't we learn that voice calls do not support the messaging service?

Reply:
Yes -- test has since been removed